### PR TITLE
Add invites to iframe API

### PR DESCRIPTION
**Pull request by hristoterezov**
WIP

### CODE REVIEW

File: modules/API/external/external_api.js [hunk 1]
```diff
@@ -200,6 +200,8 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
      * authentication.
      * @param {string} [options.onload] - The onload function that will listen
      * for iframe onload event.
+     * @param {Array<Object>} [options.invites] - Array of objects containing
```
Comment (saghul) [inline modules/API/external/external_api.js]: Can we not have this here? It's already polluted enough as it is... and the API user can trigger this themselves at any point really.

File: react/features/invite/actions.js [hunk 2]
```diff
@@ -64,3 +74,85 @@ export function updateDialInNumbers() {
...
+        }
+
+        // Sipgw calls are fire and forget. Invite them to the conference
+        // then immediately remove them from {@link invitesLeftToSend}.
+        const vrooms = invitesLeftToSend.filter(item =>
+            item.type === 'videosipgw');
+
+        conference
+            && vrooms.length > 0
+            && dispatch(inviteVideoRooms(conference, vrooms));
```
Comment (lyubomir) [inline react/features/invite/actions.js]: 1. If `dispatch(inviteVideoRooms(conference, vrooms))` is to be in `sendInvitesForItems` and
2. given that `inviteVideoRooms` is an action creator from before this PR, then
3. `sendInvitesForItems` is logically an asynchronous action creator.

Am I missing something?

**Comment by saghul** [comment]
@hristoterezov Hey, I had another look at this, and I was wrong, I apologize. I had missed the fact that `inviteVideoRooms` was an action creator itself. So it all makes sense now!

I'll take another look in the morning (as you can see, my judgement is not at its best late in the day) and merge it if all checks out (and if @lyubomir doesn't beat me to it :-) )

File: modules/API/API.js [hunk 3]
```diff
@@ -107,8 +108,24 @@ function initCommands() {
...
-    transport.on('request', ({ name }, callback) => {
+    transport.on('request', (request, callback) => {
+        const { name } = request;
+
         switch (name) {
+        case 'invite':
+            APP.store.dispatch(
+                sendInvitesForItems(request.invitees))
+                .then(failedInvites => {
+                    const failed = failedInvites.length === 0;
```
Comment (lyubomir) [inline modules/API/API.js]: `const failed = failedInvites.length === 0;` doesn't look right. I think it should be `const failed = failedInvites.length;`

**Pull request merged**